src/atomicui/molecules/IconicInfoCard/IconicInfoCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC, ReactElement } from "react";
+
+import { Flex } from "@aws-amplify/ui-react";
+import { TextEl } from "@demo/atomicui/atoms";
+
+interface IconicInfoCardProps {
+	IconComponent?: ReactElement;
+	title: string;
+	description: string;
+	textContainerMarginLeft?: string;
+	cardMargin?: string;
+	cardAlignItems?: string;
+	subDescription?: string;
+}
+
+const IconicInfoCard: FC<IconicInfoCardProps> = ({
+	IconComponent,
+	title,
+	description,
+	textContainerMarginLeft = "0.7rem",
+	cardMargin = "1rem 0",
+	cardAlignItems = "flex-start",
+	subDescription = ""
+}) => {
+	return (
+		<Flex direction="row" gap="large" margin={cardMargin} alignItems={cardAlignItems} justifyContent="flex-start">
+			{IconComponent}
+			<Flex direction="column" gap={subDescription ? 0 : "3px"} marginLeft={textContainerMarginLeft}>
+				<TextEl fontSize="1rem" variation="secondary" fontFamily="AmazonEmber-Regular" text={title} />
```

Comment:
replace TextEl with Text, font sizes and family have been added to `util.scss` can be referenced from there